src/disassembling/rzil.md

```diff
@@ -0,0 +1,151 @@
+# RzIL
+
+RzIL is the new intermediate language in Rizin, primarily intended for representing the semantics of machine code. It is designed as a clone of BAP's [Core Theory](http://binaryanalysisplatform.github.io/bap/api/master/bap-core-theory/Bap_core_theory/), with minor deviations where necessary.
+
+More details related the implementation can be found [here](https://github.com/rizinorg/rizin/blob/dev/doc/rzil.md).
+
+## IL statements
+
+IL statements for each [opcode](https://github.com/rizinorg/rizin/blob/dev/librz/include/rz_il/rz_il_opcodes.h) can be represented as a string, composed of s-expressions via command `aoi` (JSON output is available via `aoj`).
```

Comment:
I am always confused about the term `opcode`. Shouldn't it be `instruction` here?
`opcode` is for the me the bits identifying the instruction.

Reply:
you are right. the il opcode is like a microcode for an instruction. i will change this.